From ac19946f3ca80f8bd1e72d38c83635942fb07537 Mon Sep 17 00:00:00 2001
From: "wenping.zhang" <wenping.zhang@rock-chips.com>
Date: Mon, 9 Jan 2017 13:55:20 +0800
Subject: [PATCH] arm64: dts: rockchip: add usb2 phy tuning for rk3399 discrete
 vr.

add usb2 phy tuning function to enhance the usb signal for discrete vr device.

Change-Id: Iab4911610ccf488be723ce462527ecaeafe7e446
Signed-off-by: wenping.zhang <wenping.zhang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts | 8 ++++++++
 arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts | 8 ++++++++
 arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts  | 2 ++
 3 files changed, 18 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
index 052ea1757aa3..02fb90e87817 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-disvr.dts
@@ -125,6 +125,14 @@
 	};
 };
 
+&u2phy0 {
+	rockchip,u2phy-tuning;
+};
+
+&u2phy1 {
+	rockchip,u2phy-tuning;
+};
+
 /*
  * if your hardware board have two typec port, you should define
  * fusb1 and tcphy1, such as:
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
index 9deb604e64f7..9ac533c2e5bf 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-disvr.dts
@@ -132,6 +132,14 @@
 	};
 };
 
+&u2phy0 {
+	rockchip,u2phy-tuning;
+};
+
+&u2phy1 {
+	rockchip,u2phy-tuning;
+};
+
 /*
  * if your hardware board have two typec port, you should define
  * fusb1 and tcphy1, such as:
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
index 96d04e931061..92664659e237 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-disvr-android.dts
@@ -641,6 +641,7 @@
 &u2phy0 {
 	extcon = <&fusb0>;
 	status = "okay";
+	rockchip,u2phy-tuning;
 
 	u2phy0_otg: otg-port {
 		status = "okay";
@@ -654,6 +655,7 @@
 
 &u2phy1 {
 	status = "okay";
+	rockchip,u2phy-tuning;
 
 	u2phy1_otg: otg-port {
 		status = "okay";
-- 
2.35.3

